src/lib/http-agent.js

```diff
@@ -85,7 +83,25 @@ const getAgent = async ({ httpProxy, certificateFile, exit }) => {
   }
 
   const agent = new HttpsProxyAgentWithCA(opts)
-  return agent
+  response = { ...response, agent }
+  return response
+}
+
+const getAgent = async ({ httpProxy, certificateFile }) => {
+  const result = await tryGetAgent({ httpProxy, certificateFile })
```

Comment:
[pebble] By returning an empty object instead of `undefined`, we could use destructuring:

```js
  const { error, warning, message, agent } = await tryGetAgent({ httpProxy, certificateFile })
```

Reply:
Yeah.